packages/kbn-text-based-editor/src/query_history.test.tsx

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import React from 'react';
+import { EuiInMemoryTable } from '@elastic/eui';
+import { QueryHistoryAction, getTableColumns, QueryHistory, QueryColumn } from './query_history';
+import { mountWithIntl as mount } from '@kbn/test-jest-helpers';
```

Comment:
These are net-new tests. Any reason not to use RTL instead of Enzyme?

Reply:
Not at all, I was writing tests in parallel with other things and I forgot our agrrement 🙈 

Tests changed to RTL here https://github.com/elastic/kibana/pull/178302/commits/c058d0630c533a7f0385e1bc8d3d25f446574645